### DISCUSSION TITLE
Wemos verliert immer wieder die W-Lan Verbindung

### DISCUSSION THREAD

**Discussion by Kenaschon** [dewenni/ESP_Buderus_KM271#35] [General]
Hallo,

ich habe in der Nähge der Heizung einen Wlan Repeater ca. 50 cm vom Wemos D1 mini entfernt. Der Wemos verliert 1 bis 2 Mal in der Woche die Verbindung.
Das ist mir der Vorgängerversion (die mit der config-Datei) nicht passiert.

Woran kann das liegen?

Grüße
Manfred

**Comment by the78mole** [comment]
Hallo Manfred,

grundsätzlich würde ich einen Verbindungsverlust 1-2 Mal pro Woche jetzt nicht als kritisch bzw. sogar als gewöhnliches WLAN einstufen. Normalerweise würde ich das ggf. auf Störungen oder einen automatischen Kanalwechsel zurückführen, aber wenn es nachvollziehbar an der neuen Firmware liegt, dann bin ich da leider raus. Das kann vermutlich nur @dewenni nachhaltig beantworten. Vielleicht läuft ja irgendwas voll und stört den WLAN-Stack.

Grüße,
Daniel

**Reply by dewenni** [reply]
Hallo Manfred,

Welche Version hast du denn aktuell am laufen?

Ich kann dir aktuell gar nicht sagen ob bei mir zwischendurch mal die Verbindung verloren geht.

Die aktuelle Software versucht dann auch mehrfach sich neu zu verbinden und nach dem 5. gescheiterten Versuch macht der ESP dann einen Neustart.

Das passiert bei mir sehr selten. Kommt aber auch mal vor.

**Comment by Kenaschon** [comment]
SV: V3.0.1
Logamatic-Version 3.18
Logamatic-Modul: 6

Es scheint, das die Neuverbindung nicht immer klappt. Wenn der Wemos keine Verbinung bekommt, ist er am blinken wie verrückt. Ein druck auf den Reset-Knopf und alles ist wieder gut.

Ich hatte den Wemos jetzt schon mit ioBroker per "Ping" überwacht. Ich bekomme dann eine Meldung auf mein Handy, sobald die Adresse nicht mehr erreichbar ist.

Mal läuft alles wieder trotz Meldung - mal nicht.

**Reply by dewenni** [reply]
Geh mal bitte auf die neuste 3.2.1

Ich habe da im Bereich WLAN und MQTT reconnect nochmal was geändert.

**Comment by Kenaschon** [comment]
Ich muß jetzt noch mal fragen. Gibt es eine fertige "firmware.bin"?

**Reply by dewenni** [reply]
Ja, liegt unter releases ab

https://github.com/dewenni/ESP_Buderus_KM271/releases/download/v3.2.1/esp_buderus_km271_v3_2_1.bin

**Comment by Kenaschon** [comment]
Danke. Ist aufgespielt. Werde mal testen und berichten.

**Comment by Kenaschon** [comment]
Hallo,

es scheint deutlich stabiler zu sein. Bis jetzt kein Verbindungsabbruch.

Danke
